test/built-ins/JSON/isRawJSON/length.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2024 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-json.rawjson
+description: >
+  JSON.isRawJSON.length value and descriptor.
+info: |
+  JSON.isRawJSON ( text )
+
+  18 ECMAScript Standard Built-in Objects
+  Every other data property described in clauses 19 through 28 and in Annex B.2
+  has the attributes { [[Writable]]: true, [[Enumerable]]: false,
+  [[Configurable]]: true } unless otherwise specified.
```

Comment:
It would be fine with me to delete this instead of changing it, but it is not the relevant section of ECMA-262 for this test. If you decide to keep it, I'd use this frontmatter:

```
esid: sec-built-in-function-objects
info: |
    Every built-in function object, including constructors, has a *"length"*
    property whose value is a non-negative integral Number. Unless otherwise
    specified, this value is the number of required parameters shown in the
    subclause heading for the function description. Optional parameters and rest
    parameters are not included in the parameter count.

    Unless otherwise specified, the *"length"* property of a built-in function
    object has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*,
    [[Configurable]]: *true* }.
```